compiler_opt/rl/corpus.py

```diff
@@ -30,9 +32,47 @@ class ModuleSpec:
   """
   name: str
   exec_cmd: Tuple[str, ...] = ()
+  size: int = 0
 
 
-def build_modulespecs_from_datapath(
+class Corpus:
+  """Represents a corpus. Comes along with some utility functions."""
+
+  def __init__(self,
+               data_path: str,
+               additional_flags: Tuple[str, ...] = (),
+               delete_flags: Tuple[str, ...] = (),
+               module_specs: Optional[List[ModuleSpec]] = None):
+    if module_specs is not None:
+      self._module_specs = module_specs
+    else:
+      self._module_specs = _build_modulespecs_from_datapath(
+          data_path=data_path,
+          additional_flags=additional_flags,
+          delete_flags=delete_flags)
+    self.root_dir = data_path
+    self._module_specs.sort(key=lambda m: m.size, reverse=True)
```

Comment:
why do sorting here?

Reply:
was an artifact from development, but it's useful for the bucketing as well so will be left as-is